akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -447,7 +447,7 @@ class ClientServerSpec extends WordSpec with Matchers with BeforeAndAfterAll wit
         val serverInSub = serverIn.expectSubscription()
```

Comment:
@ktoso I have added a line comment above clarifying that the setting of `request-timeout` to `infinite` in line 28 of `TimeoutDirectivesExamplesSpec` before this PR would result in failed test. However, emboldened by your comment, I will add the opposite case (finite timeout, as was previously set) to pinpoint the fact, that now the setting is symmetric - works for both cases. The side effect of this change, which resulted in change in other specs, is that the `request-timeout` is never disabled in the routing layer but just set to infinite value. So we are changing the meaning of `request-timeout=infinite` from (special) _disable in routing layer_ to (plain) _set the timeout to infinite_.